e2e-tests/trade-finance-manager/cypress/e2e/journeys/case/underwriting/underwriter-managers-decision.spec.js

```diff
@@ -61,45 +61,146 @@ context('Case Underwriting - Underwriter Manager\'s decision', () => {
+      pages.managersDecisionPage.commentsInputDecline().should('be.visible');
+    });
+
+    it('should throw validation error if no comment provided and persists radio selection', () => {
+      pages.managersDecisionPage.submitButton().click();
+
+      // radio should be selected
+      pages.managersDecisionPage.decisionRadioInputDecline().should('be.checked');
+
+      // assert errors are displayed
+      pages.managersDecisionPage.errorSummaryItems().should('have.length', 1);
+      pages.managersDecisionPage.commentsInputDeclineValidationError().should('be.visible');
+    });
+
+    it('should throw validation error if decline comment is too long', () => {
+      pages.managersDecisionPage.commentsInputDecline().type('a'.repeat(8001));
+
+      pages.managersDecisionPage.submitButton().click();
+
+      pages.managersDecisionPage.errorSummaryItems().should('have.length', 1);
+      pages.managersDecisionPage.commentsInputDeclineValidationError().should('be.visible');
+    });
+
+    it('should throw validation error if decline comment is whitespace', () => {
+      pages.managersDecisionPage.commentsInputDecline().type('      ');
+
+      pages.managersDecisionPage.submitButton().click();
+
+      pages.managersDecisionPage.errorSummaryItems().should('have.length', 1);
+      pages.managersDecisionPage.commentsInputDeclineValidationError().should('be.visible');
+    });
+  });
+});
+
+context('Case Underwriting - Underwriter Manager\'s decision - Submit Form', () => {
+  let dealId;
+  const dealFacilities = [];
+
+  before(() => {
+    cy.insertOneDeal(MOCK_DEAL_MIA, MOCK_MAKER_TFM).then((insertedDeal) => {
+      dealId = insertedDeal._id;
+
+      const { dealType, mockFacilities } = MOCK_DEAL_MIA;
+
+      cy.createFacilities(dealId, mockFacilities, MOCK_MAKER_TFM).then((createdFacilities) => {
+        dealFacilities.push(...createdFacilities);
+      });
+
+      cy.submitDeal(dealId, dealType);
+    });
+  });
+
+  beforeEach(() => {
+    cy.login(UNDERWRITER_MANAGER_1);
+    cy.visit(relative(`/case/${dealId}/deal`));
+
+    // go to underwriter managers decision page
+    partials.caseSubNavigation.underwritingLink().click();
+  });
+
+  after(() => {
+    cy.deleteDeals(dealId, ADMIN_LOGIN);
+    dealFacilities.forEach((facility) => {
+      cy.deleteFacility(facility._id, MOCK_MAKER_TFM);
+    });
   });
 
-  it('after valid form submit, displays submitted values and updates deal stage', () => {
+  it('after valid form submit, cleans and displays submitted values and updates deal stage', () => {
     pages.underwritingPage.addUnderwriterManagerDecisionButton().click({ force: true });
 
-    const MOCK_COMMENTS = 'Testing';
-    const MOCK_INTERNAL_COMMENTS = 'Internal comment';
+    const MOCK_COMMENTS = 'Approval comment. <div>Div contents</div><script>Script contents</script>';
+    const MOCK_INTERNAL_COMMENTS = 'Internal comment. <div>Div contents</div><script>Script contents</script>';
```

Comment:
Minor adjustment to the `MOCK_COMMENTS` can we use this please to test the robustness of the `xss()`.

```
Approval comment. <div>Div contents</div><script>Script contents</script> &lt;img src = 'data:abc' /&gt;
```

Reply:
I tested this out and the `&lt;img src = 'data:abc' /&gt` doesn't get stripped out.

Is it worth including it in this assertion to show that it doesn't get stripped out, or should I leave the test just showing what does get stripped out?

Reply:
Please leave it in the assertion and show that it doesn't get stripped out, I would expect it not to be stripped out and rendered as `<` or `>`

Reply:
Will do. As confirmed on Slack I will make this change and merge this pull request.